### ISSUE TITLE
When updating the flatpak goldwarden gets stuck in a loop asking me to unlock the vault

### ISSUE THREAD

**Issue by MostHated** [quexten/goldwarden#93]
Hey there,
I have an update script that runs when I type `up` , which runs apt update && apt upgrade, then flatpak update, followed by checking dotnet tools, etc, etc. 

During this process presumably during the `flatpak update`, goldwarden gets stuck in a loop asking me to unlock the vault with my pin, which I do, but the prompt keeps repeating regardless of if I enter the pin and hit the confirmation button, or if I hit cancel, the input modal will come right back.

After a dozen+ tries, I got frustrated and just held `enter` for a few, and it went away, and I briefly saw a notification that Goldwarden had hit a rate limit against my vaultwarden. It did at least let me go on about my business.
I don't remember the words exactly that it said, I only saw them for a second and I didn't see it in syslog, and I can't seem to locate any goldwarden specific logs? 

I will keep looking for them and once found I will post them. Are they located somewhere out of the ordinary? I don't see them in `~/.var/app/com.quexten.Goldwarden` or `~/.local/share/flatpak`, and not in `/var/lib/flatpak/app/com.quexten.Goldwarden` unless I missed them.

**Comment by quexten** [comment]
Oh interesting, I'm not entirely sure why it would ask you to unlock your vault during update. Do you use the SSH agent? 

Anyways, the logs are currently only printed, so for the systemd service you can see them via journalctl, in flatpak you have to restart the app with "flatpak run" and watch the logs. I guess this could be changed to also write them to disk in both cases.

**Comment by MostHated** [comment]
Yeah, I would say to always to disk would be ideal, as I could have just posted logs in my initial post. The only time I would do flatpak run is if I *know* there is an error, and how to reproduce it. 

With this, for example, I run my update script now, nothing out of the ordinary happens anymore after the first time that I force it to rate limit. So I am not sure what that did/cause as a result, and trying to do it again does nothing. I am guessing that it only reoccurs on a restart or logout, etc. This is not the first time it has happened, but after it happened before, I had to get back to work and forgot about it until it happened again, which is where we are at the moment.

As for an ssh-agent in reference to goldwarden, would I have had to set something additional up for that? If so, then no, I don't believe so. The only thing I have done with it so far was setup the hot key, but with having to input my password each time, and then hit approve on a second prompt to allow listing credentials (which I just tried again to make sure, and now there is no listing when I try and search, so I will have to kill it and rerun it with run command). I found it was more steps than just using my local bitwarden client. There may be a way to make this require less, and/or work everytime, but I have not looked into it.
